core-processor/src/ext.rs

```diff
@@ -503,6 +505,24 @@ impl EnvExt for Ext {
 
         self.charge_sending_fee(delay)?;
 
+        if delay != 0 {
+            // Take delay and get cost of block
+            // Calculate reserve like  wait_cost * (delay + reserve_for)
+            // Mikita todo
+            let cost_per_block = self.context.dispatch_hold_cost;
+            let waiting_reserve = u64::from(
```

Comment:
Calculate in `u64`. Don't convert `u32` into `u64`

Reply:
done